server/tsconfig.paths.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
kan je dit in een andere pr doen? heeft niets met tslint te maken en weet niet of ik hier wel zo fan van ben (hou dan liever die discussie apart, dan zit deze pr niet geblokkeerd)